tests/trustless_gateway_car_test.go

```diff
@@ -39,6 +42,7 @@ func TestTrustlessCarPathing(t *testing.T) {
 						Exactly().
 						InThatOrder(),
 				),
+			IPIP: IPIP402,
```

Comment:
We can associate an IPIP to a test, later maybe we can associate to the check itself.